.github/ISSUE_TEMPLATE/support-ticket.yml

```diff
@@ -0,0 +1,85 @@
+name: Support Ticket
+description: Support ticket for requesting help from Boost team
+title: "[Support Ticket]: "
+labels: ["support", "triage"]
+body:
+- type: checkboxes
+  attributes:
+    label: Boost component
+    description: Please select the boost component you are filing a bug for
```

Comment:
Just so I understand, what is the rationale for having this option at all? 

It will take very little time for us to label it once the issue is captured, not having it would reduce the barrier for capturing an issue, and we need to read through the issue to check it's labeled right in any case. No?

Reply:
There are multiple way to creating a support ticket. But, generally agreed upon industry standard is to collect some basic information to allow support engineers to provide an initial direction of investigation rather than asking same questions from the client/customer. 
If we don't even collect which component would is impacted, then this template would be no different than slack conversations. 
PS: There labels are not github labels.